vendor/awsping/Makefile

```
@@ -0,0 +1,17 @@
+include ../../tasks/Makefile.package
+include ../../tasks/Makefile.apk
+
+# Package details
+export VENDOR ?= ekalinin
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/$(PACKAGE_VERSION)/$(PACKAGE_NAME).$(OS).$(ARCH).tgz
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+
+install:
+	$(CURL) -o - $(DOWNLOAD_URL) | tar --wildcards -zxO awsping > $(INSTALL_PATH)/awsping
+	chmod +x $(INSTALL_PATH)/awsping
+
+test:
+	$(PACKAGE_EXE) --version
```

@osterman 
not sure if `--version` is the correct command.

https://github.com/ekalinin/awsping/blob/master/main.go#L26

uses `v` and the module does not use any CLI libraries, but just parses the arguments directly.
```suggestion
	$(PACKAGE_EXE) -v
```